projects/lessgas/index.js

```diff
@@ -0,0 +1,74 @@
+const sdk = require("@defillama/sdk");
+const axios = require('axios')
+
+const BigNumber = require("bignumber.js");
+const HOST='https://api2.lessgas.xyz'
+// const HOST='http://127.0.0.1:9912'
+const request_staking = async () => {
+  let config = {
+    method: 'get',
+    maxBodyLength: Infinity,
+    url: `${HOST}/llama/staking`,
+  };
+  let response =  await axios.request(config)
+  // console.log(response.data);
+  let items=response.data.data.result;
+  let total=0;
+  for (const item of items) {
+    try {
+      let { output } = await sdk.api.abi.call({
+        abi: "erc20:balanceOf",
+        target: item[1],
+        params: [item[0]],
+        chain: "map"
+        // block: chainBlocks["arbitrum"],
+      });
+      console.log(output);
+     total =  new BigNumber(total).plus(new BigNumber(output).div(10**18).multipliedBy(new BigNumber(item[2])));
```

Comment:
I am talking about this line, item[2] here is the price right? we use our own api for pulling token price, for example xcCGF on moonbeam: https://coins.llama.fi/prices/current/moonbeam:0xffffffff44bd9d2ffee20b25d1cf9e78edb6eae3

We dont have a token list for map tokens, so currently it is returning 0

Reply:
do you know of an official bridge, that I can use to map map tokens to brc20 on bitcoin, and/or what tokens on map  chain are listed on coingecko? without these info we cant price tokens on map